packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -288,6 +288,12 @@ export interface ExpoConfig {
      * Experimentally enable React Server Functions support in Expo CLI and Expo Router.
      */
     reactServerFunctions?: boolean;
+    /**
+     * Experimentally enable downloading cached builds from remote.
+     */
+    remoteBuildCache?: {
+      provider: 'eas';
+    };
```

Comment:
this will need to be added to the schema on universe https://github.com/expo/universe/blob/main/server/www/xdl-schemas/UNVERSIONED-schema.json#L1491

Reply:
Yep, here it is https://github.com/expo/universe/pull/19600